sections/03_categorize.md

```diff
@@ -127,83 +132,88 @@ repurpose features from task to task, improving overall predictions as the field
+domain. However, numerous challenges including data integration (patient
+demographics, family history, laboratory tests, text-based patient records,
+image analysis, genomic data) and better handling of streaming temporal data
+with many features, will need to be overcome before we can fully assess the
+potential of deep learning for this application area.
 
 ##### Opportunities
 
 However, significant work needs to be done to move these from conceptual
 advances to practical game-changers.
 
 * Large data resources (see sample # issues that mammography researchers are
-    working around)
-* Semi-supervised methods to take advantage of large number of unlabeled
-    examples
-* Transfer learning.
+working around) * Semi-supervised methods to take advantage of large number of
+unlabeled     examples * Transfer learning.
 
 ##### Unique challenges
 
 Additionally, unique barriers exist in this space that may hinder progress in
 this field.
 
-###### Data sharing and privacy?
+###### Standardization/integration
 
-*This is clearly a big issue. We should at least mention it. Deep learning likes
-lots of data, and sharing restrictions don't allow that. Perhaps a paragraph on
-current best practices and how they relate to deep learning. A lack of data (due
-to privacy and sharing restrictions) may hamper deep learning's utility in this
-area in ways that it doesn't for image analysis, etc. Perhaps this will be the
-Achilles heal of deep learning in this area. A couple things to think about
-[doi: 10.1126/science.1229566 doi:10.1016/j.cels.2016.04.013]*
+EHRs are designed and optimized primarily for patient care and billing purposes,
+meaning research is at most a tertiary priority. This presents significant
+challenges to EHR based research in general, and particularly to data
+intensive deep learning research. EHRs are used differently even within the same
+health care system (@pmid:PMC3797550, @pmid:PMC3041534). Individual users have unique usage patterns, and different departments have different priorities which introduce missing data in a non-random fashion. Just et al. demonstrated that even the most
+basic task of matching patients can be challenging due to data entry issues
+[@pmid:27134610]. This is before considering challenges caused by system
+migrations and health care system expansions through acquisitions. Replication
+between hospital systems requires controlling for both these systematic biases as
+well as for population and demographic effects.  Historically, rules-based algorithms have been popular but because these are developed at a single institution and trained with a specific patient population they do not transfer easily to other populations [@doi:10.1136/amiajnl-2013-001935 ]. Wiley et al. [@doi:10.1142/9789813207813_0050] showed that warfarin dosing algorithms often under perform in African Americans, illustrating that some of these issues are unsolved even at a treatment best practices level. 
```

Comment:
I like where this discusses rules-based approaches. Can you tie it to ML-based methods? Is there a parallel here with regard to something like translation where rules-based methods are now being superseded by deep learning methods?